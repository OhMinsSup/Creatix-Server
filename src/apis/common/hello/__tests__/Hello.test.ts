import faker from 'faker';
import resolvers from '../Hello.resolvers';

const username = faker.internet.userName();

describe('Hello GraphQL', () => {
  test('Hello World Not Value Output', async () => {
    expect(
      await resolvers.Query.Hello(
        {},
        {
          name: ''
        },
        {},
        {}
      )
    ).toMatchSnapshot();
  });

  test('Hello World Using Value Output', async () => {
    expect(
      await resolvers.Query.Hello(
        {},
        {
          name: username
        },
        {},
        {}
      )
    ).toMatchSnapshot();
  });
});
