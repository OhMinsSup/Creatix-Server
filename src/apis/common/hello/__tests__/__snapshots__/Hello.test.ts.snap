// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello GraphQL Hello World Not Value Output 1`] = `
Object {
  "result": "Hello World",
}
`;

exports[`Hello GraphQL Hello World Using Value Output 1`] = `
Object {
  "result": "Hello Eliane.Mueller",
}
`;
